docs/eog/case-studies-cw20-budget.rst

```diff
@@ -0,0 +1,29 @@
+.. _CW20-Budget: 
+
+CW20 Budget
+=============
+
+We were extremely lucky that there were no additional costs for moving the event online - the University of Manchester (UoM) had already provided staff with Education licensed Zoom accounts which allowed for unlimited meeting duration for up to 300 people. 
+There were enough UoM staff members on the CW20 delivery team that we had enough Zoom Host accounts for the main plenary room plus 4 additional rooms in parallel for the Mini-workshop and demo sessions. 
+
+We did reduce the cost of registration from £250 to £50 as there would be no catering involved in an online event. 
+The reasons we didn't make the event free included the staff time that had gone into organising the event, to prevent a large number of no-shows on the day and because there were prizes involved in the Hack Day.
```

Comment:
```suggestion
The reasons we didn't make the event free included the staff time that had gone into organising the event, to prevent a large number of no-shows on the day and because there were prizes involved.
```

Reply:
I took Hack Day out as it also applied to Collaborative Ideas and you mentioned it further down the page.